app/templates/measure-concepts/edit.hbs

```diff
@@ -13,18 +13,23 @@
     class="au-c-form au-o-layout au-u-padding-top-large"
     {{on "submit" this.editMeasureConcept}}
   >
-    <p>
-      <AuLabel for="description">{{t "measureConcept.attr.description"}}</AuLabel>
+  {{#let (is-empty measureConcept.description) as |isInvalid|}}
+      <p>
```

Comment:
The indentation is funky again in some places. Can you double check your editor formatter and/or disable it? :smile:.